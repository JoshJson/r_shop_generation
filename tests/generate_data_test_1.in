Дни	Поставки
1	17
2	14
3	16
4	19
5	13
6	12
7	15
8	13
9	12
10	12
11	11
12	19
13	16
14	16
15	18
