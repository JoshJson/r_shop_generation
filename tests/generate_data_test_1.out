Дни	Продажи
1	15
2	14
3	9
4	10
5	11
6	11
7	9
8	9
9	11
10	9
11	11
12	16
13	9
14	12
15	13
